index.html

```diff
@@ -809,6 +809,21 @@ <h2 id="collections">Collection Functions (Arrays or Objects)</h2>
 =&gt; [[1, 3, 5], [0, 2, 4]]
 </pre>
 
+      <p id="isEqual">
+        <b class="header">isEqual</b><code>_.isEqual(list, other)</code>
+        <br />
+        Performs an optimized deep comparison between one <b>list</b> and the <b>other</b>, to determine
```

Comment:
How about something like this, under *Utility*?
> **isEqual** `_.isEqual(first, second)`
Performs a comparison between **first** and **second** of arbitrary types, without type conversion. Where applicable, performs an optimized deep comparison to determine if **first** and **second** should be considered equal.
```
var stooge = {name: 'moe', luckyNumbers: [13, 27, 34]};
var clone  = {name: 'moe', luckyNumbers: [13, 27, 34]};
stooge == clone;
=> false
_.isEqual(stooge, clone);
=> true
```